src/sortStudents.ts

```diff
@@ -1,16 +1,75 @@
-
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
+}
+
+export enum SortOrder {
+  Asc = 'asc',
+  Desc = 'desc',
 }
 
-// create SortOrder type
-export type SortOrder;
+function getAverageGrade(student: Student): number {
+  return student.grades.reduce((total, grade) => total + grade)
+    / student.grades.length;
+}
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const studentCopy: Student[] = [...students];
+
+  switch (sortBy) {
+    case SortType.Name:
+    case SortType.Surname:
+      studentCopy.sort((firstStudent, secondStudent): number => {
```

Comment:
```suggestion
      return studentCopy.sort((firstStudent, secondStudent): number => {
```
You can immediately return result.